src/imports/envo_terms.txt

```diff
@@ -94,4 +94,11 @@ http://purl.obolibrary.org/obo/ENVO_00000006 # second-order administrative regio
 http://purl.obolibrary.org/obo/ENVO_00000005 # first-order administrative region
 http://purl.obolibrary.org/obo/ENVO_01000930 # ecological corridor
 http://purl.obolibrary.org/obo/ENVO_01000951 # natural environment
-http://purl.obolibrary.org/obo/OBI_0000245 # organization
+http://purl.obolibrary.org/obo/ENVO_02500026 # organization
+http://purl.obolibrary.org/obo/ENVO_02500042 # pollution control
+http://purl.obolibrary.org/obo/ENVO_01001170 # active ecosystem management
+http://purl.obolibrary.org/obo/ENVO_02500043 # pollution monitoring
+http://purl.obolibrary.org/obo/ENVO_01000906 # aquaculture process
+http://purl.obolibrary.org/obo/ENVO_01001442 # agriculture
+http://purl.obolibrary.org/obo/ENVO_01001451 # usage of an environment for business
+http://purl.obolibrary.org/obo/FOODON_03470107 # food preservation process
```

Comment:
Careful - only use the terms.txt file to import terms from the named ontology. This line will import ENVO's import of FOODON's term, which creates a potentially (inevitably) messy dependency chain.

Reply:
Yes, I understand now. Should I remove the foodon term?